wgpu-core/src/command/render.rs

```diff
@@ -1262,6 +1275,29 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         raw.draw_indexed_indirect(&buffer.raw, offset, 1, 0);
                     }
                 }
+                RenderCommand::PushDebugGroup {
+                    color,
+                    len,
+                    phantom_marker,
+                } => unsafe {
+                    let (new_peeker, label) =
+                        { phantom_marker.decode_unaligned(peeker, len, raw_data_end) };
+                    peeker = new_peeker;
+
+                    raw.begin_debug_marker(std::str::from_utf8(label).unwrap(), color)
```

Comment:
nit: we should include `std::str` at the top of the module to be consistent with other uses of std